### PR TITLE
Add event code documentation and clean up event code

### PR DESCRIPTION
**Pull request by kimlisa**
#### Description
I noticed gravity had a documentation on how their gravity event code was formulated. I initially tried to sync teleport with their pattern, but teleport deviates from their pattern too much. So I just added a documentation on the current pattern so it stays consistent.

### CODE REVIEW

File: lib/events/codes.go [hunk 1]
```diff
@@ -249,7 +249,17 @@ var (
 	}
 )
 
-// OSS event codes start with "T".
+// There is no strict algorithm for picking an event code, however existing
+// event codes are currently loosely categorized as follows:
+//
+//  * OSS event codes start with "T" and belongs in this const block.
```
Comment (webvictim) [inline lib/events/codes.go:255]: belongs -> belong

File: lib/events/codes.go [hunk 2]
```diff
@@ -249,7 +249,17 @@ var (
...
+//
+//  * Enterprise event codes start with "TE", and is in another const block.
```
Comment (webvictim) [inline lib/events/codes.go:257]: is in -> are in

**Review by webvictim — COMMENTED** [review]
This looks OK to me. Not sure if changing the codes affects anything downstream though (like end-users who might be parsing these codes)

File: lib/events/codes.go (hunk 1 above)
Comment (kimlisa) [reply lib/events/codes.go:255]: revised

File: lib/events/codes.go (hunk 2 above)
Comment (kimlisa) [reply lib/events/codes.go:257]: revised

**Comment by kimlisa** [comment]
> This looks OK to me. Not sure if changing the codes affects anything downstream though (like end-users who might be parsing these codes)

@webvictim the codes I changed were the ones I recently added in a previous PR

**Review by webvictim — APPROVED** [review]

**Review by awly — APPROVED** [review]

**Pull request merged**